cmd/generate.go

```diff
@@ -132,6 +138,10 @@ func createProject(sourcePath string) (*AtlantisProject, error) {
 	if err != nil {
 		return nil, err
 	}
+	// if dependecies AND err is nil then return nils to indicate we should skip this project
+	if err == nil && dependencies == nil {
```

Comment:
I may very well be wrong here, but wouldn't we still want to add this project to the atlantis.yaml file so that we could have it's autoplan setting set?

Reply:
If you're concerned that atlantis will autodetect the file the answer is no. Well the _short_ answer is no 😅 

I've noticed there's some confusion around the "autoplan" feature and the unnamed feature where atlantis automatically detects projects to plan.  Basically "autoplan" refers _only_ to automatically running a plan against a project.  [The automatic detection of projects to plan is disabled by simply defining an *atlantis.yaml*](https://www.runatlantis.io/docs/repo-level-atlantis-yaml.html#enabling-atlantis-yaml):
> Once an atlantis.yaml file exists in a repo, Atlantis won't try to determine where to run plan automatically. Instead it will just follow the project configuration.

Reply:
wonderful, thanks for the update!